backend/tests/v5/unit/services/eventsListener/eventsListener.test.js

```diff
@@ -607,6 +608,52 @@ const testModelEventsListener = () => {
 			);
 		};
 
+		const importTicketsTest = async (isFederation) => {
+			const waitOnEvent = eventTriggeredPromise(events.TICKETS_IMPORTED);
+			const template = generateTemplate();
+			const type = generateRandomString();
+			const data = {
+				teamspace: generateRandomString(),
+				project: generateRandomString(),
+				model: generateRandomString(),
+				tickets: times(10, () => ({
+					type,
+					...generateRandomObject(),
+				})),
+			};
+
+			TicketTemplates.getTemplateById.mockResolvedValueOnce(template);
+			ModelSettings.isFederation.mockResolvedValueOnce(isFederation);
+			const event = isFederation ? chatEvents.FEDERATION_NEW_TICKET : chatEvents.CONTAINER_NEW_TICKET;
+			EventsManager.publish(events.TICKETS_IMPORTED, data);
+			expect(ModelSettings.isFederation).toHaveBeenCalledTimes(1);
+			expect(ModelSettings.isFederation).toHaveBeenCalledWith(data.teamspace, data.model);
+
+			expect(TicketLogs.addImportedLogs).toHaveBeenCalledTimes(1);
+			expect(TicketLogs.addImportedLogs).toHaveBeenCalledWith(
+				data.teamspace,
+				data.project,
+				data.model,
+				data.tickets,
+			);
+
+			await waitOnEvent;
+
+			expect(TicketTemplates.getTemplateById).toHaveBeenCalledTimes(1);
+			expect(TicketTemplates.getTemplateById).toHaveBeenCalledWith(data.teamspace, type);
+			expect(ChatService.createModelMessage).toHaveBeenCalledTimes(data.tickets.length);
+
+			data.tickets.forEach((ticket) => {
```

Comment:
They're not being mocked right now, so we're calling the real function and getting the real data.

I just tried it quickly and mocking it will result in incorrect data so it kind of makes things worse.

Questionnable if we want to do it really, but I think it's out of scope for this issue since It's just a copy of the existing NEW_TICKET implementation.